TOOLS/ce/src/main/scala/org.opalj.ce/Comment.scala

```diff
@@ -0,0 +1,144 @@
+                } else if (line.trim.startsWith("@brief")) {
+                    brief = line.trim.stripPrefix("@brief").trim
+                } else if (line.trim.startsWith("@constraint")) {
+                    constraints.addOne(line.trim.stripPrefix("@constraint").trim)
+                } else if (line.trim.startsWith("@type")) {
+                    datatype = line.trim.stripPrefix("@type").trim
+                } else {
+                    description.addOne(line.trim.stripPrefix("@description").trim)
+                }
+            }
+            commentsCommitted = true
+        }
+    }
+
+    /**
+     * Converts the Comment object into HTML syntax.
+     * Commits the Comments if it has not happened previously.
+     * @return returns the entire object as HTML code ready for insertion into an HTML file.
+     */
+    def toHTML: String = {
+        if (!commentsCommitted) commitComments()
+        var HTMLString = ""
+        if (!this.isEmpty) {
+            if (description.nonEmpty) {
+                HTMLString += "<p><b> Description: </b> <br>"
+                for (line <- description) {
+                    HTMLString += line + "<br>"
+                }
+                HTMLString += "</p>"
+            }
+            if (datatype.nonEmpty) HTMLString += "<p><b> Type: </b>" + datatype + "<br></p>"
+            if (constraints.nonEmpty) {
+                if (datatype.equals("enum")) {
+                    HTMLString += "<p><b> Allowed Values: </b><br>"
+                } else {
+                    HTMLString += "<p><b> Constraints: </b><br>"
+                }
+                for (line <- constraints) {
+                    HTMLString += line + "<br>"
+                }
+                HTMLString += "</p>"
+            }
+        }
+        HTMLString
+    }
+
+    /**
+     * Merges another committed comment into this comment.
+     * @param comment accepts the comment that should be merged into this comment.
+     */
+    def mergeComment(comment: Comment): Unit = {
+        // Make sure every comment is committed
+        this.commitComments()
+        comment.commitComments()
+
+        // Merge comments
+        if (this.label != "" || comment.label != "") {
+            this.label = comment.label + "." + this.label
+        }
+        this.brief = comment.brief + "   " + this.brief
+        this.brief = this.brief.trim
+        this.description.addAll(comment.description)
+        this.constraints.addAll(comment.constraints)
+    }
+
+    /**
+     * Checks if the comment is empty.
+     * @return returns true if the comment is empty but the label property (the label property is set automatically for config files.).
+     */
+    def isEmpty: Boolean = {
+        if (commentBuffer.length <= 0) return true
+        if (commentBuffer.length <= 1 && label.trim != "") return true
+        false
+    }
+
+    def getBrief: String = {
+        if (this.brief.isEmpty) {
```

Comment:
Sorry, habit from the Java times. Since I had zero variable name conflicts within the entire subproject, I was able to remove every single "this" from ConfigurationExplorer